appveyor.yml

```diff
@@ -41,6 +41,7 @@ before_build:
 build: on
 
 build_script:
+  - set CL=/MP
```

Comment:
So, Shall I remove `set cores=%NUMBER_OF_PROCESSORS%` ?

Reply:
I don't know. If it's the same, sure.